core/store/orm/config.go

```diff
@@ -202,6 +205,15 @@ func init() {
 		OptimismGasFees:                  true,
 	}
 
+	// Fantom
+	fantomTestnet := mainnet
+	fantomTestnet.EthGasPriceDefault = *big.NewInt(15000000000)
+	fantomTestnet.EthMaxGasPriceWei = *big.NewInt(100000000000)
+	fantomTestnet.MinIncomingConfirmations = 3
+	fantomTestnet.MinRequiredOutgoingConfirmations = 2
+	fantomTestnet.MinimumContractPayment = assets.NewLink(3500000000000000)
+	fantomMainnet := fantomTestnet
```

Comment:
Is EthGasLimitDefault still set correctly for every other chain?

We should probably have tests on this.

Reply:
https://github.com/smartcontractkit/chainlink/pull/4467/files#diff-64548e32f60e2f8e2be0f485426f7c13db905f8947ed5285cb15816b87a6a215R44

Reply:
Yup saw that, I meant we should probably have tedious, boring and exhaustive tests on every variable for every chain.

Reply:
So, for example, Optimism does not have EthGasLimitDefault set. Will it do the correct thing?

Reply:
Added a table test.